### PR TITLE
Update versioning.mdx

### PR DESCRIPTION
**Pull request by ZoeAde**
Adding clarity about what happens if an organization doesn't update an SDK past EOL

### CODE REVIEW

**Comment by ZoeAde** [comment]
@bwoskow-ld - I wasn't able to tag you as a reviewer directly (GH permissions...) but let me know if you have any feedback on this.

File: src/content/topics/sdk/concepts/versioning.mdx [hunk 1]
```diff
@@ -85,7 +85,7 @@ Examples of breaking changes that would qualify for a major version release incl
 
 ## Supported versions
 
-LaunchDarkly SDKs follow LaunchDarkly's end of life (EOL) policy to determine each SDK version's maintenance window. The end of life policy also defines when LaunchDarkly SDKs can drop support for older versions of their underlying platforms.
+LaunchDarkly SDKs follow LaunchDarkly's end of life (EOL) policy to determine each SDK version's maintenance window. The end of life policy also defines when LaunchDarkly SDKs can drop support for older versions of their underlying platforms. We cannot make any guarantees around continuing to support SDK versions that are past end of life. These versions certainly may continue to function, and in many cases they do, however the LaunchDarkly service may not continue to fully support the protocols used by these older SDK versions.
```
Comment (bwoskow-ld) [inline src/content/topics/sdk/concepts/versioning.mdx:88]:
```suggestion
LaunchDarkly SDKs follow LaunchDarkly's end of life (EOL) policy to determine each SDK version's maintenance window. The end of life policy also defines when LaunchDarkly SDKs can drop support for older versions of their underlying platforms. We do not make any guarantees around continuing to support SDK versions that are past end of life. These versions certainly may continue to function, and in many cases they do, however the LaunchDarkly service may not continue to fully support the protocols used by these older SDK versions.
```

**Review by bwoskow-ld — COMMENTED** [review]
Minor language change, looks good otherwise

File: src/content/topics/sdk/concepts/versioning.mdx (hunk 1 above)
Comment (scribblingfox) [reply src/content/topics/sdk/concepts/versioning.mdx:88]: Agree with Ben's edit.

**Review by mmrj — APPROVED** [review]
LGTM with Ben's suggestion. 
@ZoeAde let me know if you have any issues with updating / merging / GitHub permissions.

**Review by scribblingfox — APPROVED** [review]
Approving this contingent on Ben's edit being accepted.

**Pull request merged**